app/pb/threads.proto

```diff
@@ -4,6 +4,14 @@ package org.couchers.api.threads;
 
 import "google/protobuf/timestamp.proto";
 
+// object returned by pages, discussions etc to refer to top level threads.
+message TopLevelThread {
+  // top level thread id.
+  int64 thread_id = 1;
+  // total number of comments and replies to comments.
+  int64 num_responses = 2;
+}
+
```

Comment:
Move this under the service definition.

I would also just call it `Thread` and specify it's top level in a comment